test/notebooks/notebookTestUtils.ts

```diff
@@ -0,0 +1,90 @@
+import { ElementHandle, Frame } from "puppeteer";
+import { TestExplorerParams } from "../../src/TestExplorerParams";
+
+export const NOTEBOOK_OPERATION_DELAY = 5000;
+export const RENDER_DELAY = 1000;
+
+let testExplorerFrame: Frame;
+export const getTestExplorerFrame = async (): Promise<Frame> => {
+  if (testExplorerFrame) {
+    return testExplorerFrame;
+  }
+
+  const notebooksTestRunnerApplicationId = process.env.NOTEBOOKS_TEST_RUNNER_TENANT_ID;
```

Comment:
Where are all these `process.env` variables expected to be loaded from? Do they need to be added to .env.example so they can be run locally?

Reply:
Yes, they need to be injected in the environment. WIll update .env.example file